assets/digitalocean/files/opt/teleport/start_teleport.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+clear
+echo
+cat <<EOM
+*********************************************************
+**              Configuring Teleport                   **
+*********************************************************
+EOM
+
+# ask for cluster name
+echo "___"
+printf "• Enter Teleport cluster name (public domain name): "
+read -r cluster_name
+
```

Comment:
It would be good to have a step to tell the user to create an `A` record for the Teleport instance. 

e.g. Create an A record for `cluster_name` that points to [ip address of droplet].